product_docs/docs/pem/9/pem_upgrade/moving_pem_server.mdx

```diff
@@ -368,3 +368,18 @@ Before starting the server migration, make sure that the firewalls between the s
     After modifying the registry, you must restart the PEM agent's service. You can use the Services dialog box, accessed through the Windows Control Panel, to restart the `Postgres Enterprise Manager - pemagent` service.
 
     After moving the server, change the connection properties in any installed PEM clients to connect to the new host of the PEM server, agents, and monitored servers.
+
+!!! Note
```

Comment:
Does this only apply to Windows? If not, I'd create a new section titled: "Troubleshooting issues after moving the server" or something along those lines.